api/src/prescription/campaign/domain/usecases/find-paginated-filtered-organization-campaigns.js

```diff
@@ -1,11 +1,44 @@
-const findPaginatedFilteredOrganizationCampaigns = function ({
+import { withTransaction } from '../../../../shared/domain/DomainTransaction.js';
+import { CampaignResultLevelsPerTubesAndCompetences } from '../models/CampaignResultLevelsPerTubesAndCompetences.js';
+
+const findPaginatedFilteredOrganizationCampaigns = withTransaction(async function ({
+  locale,
   userId,
   organizationId,
   filter,
   page,
   campaignReportRepository,
+  campaignParticipationRepository,
+  learningContentRepository,
+  knowledgeElementSnapshotRepository,
+  withCoverRate,
 }) {
-  return campaignReportRepository.findPaginatedFilteredByOrganizationId({ organizationId, filter, page, userId });
-};
+  const campaignReports = await campaignReportRepository.findPaginatedFilteredByOrganizationId({
+    organizationId,
+    filter,
+    page,
+    userId,
+  });
+
+  if (withCoverRate) {
```

Comment:
Je pense qu'on peut privilégier un early return, pour éviter d'avoir un if et une imbrication tout le long de la fonction : 

```suggestion
  if (!withCoverRate) {
  	return campaignReports;
  }
```